ios/Classes/PTFlutterViewController.h

```diff
@@ -15,19 +15,18 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic) BOOL documentLoaded;
 
 // viewer options
-@property (nonatomic, assign, getter=isReadOnly, setter=setReadOnly:) BOOL readOnly;
-@property (nonatomic, assign, getter=isThumbnailViewEditingEnabled, setter=setThumbnailViewEditingEnabled:) BOOL thumbnailViewEditingEnabled;
-@property (nonatomic, copy, getter=getAnnotationAuthor) NSString * annotationAuthor;
-@property (nonatomic, assign, getter=isContinuousAnnotationEditing, setter=setContinuousAnnotationEditing:) BOOL continuousAnnotationEditing;
-
-- (void)setAnnotationAuthor:(NSString*)annotationAuthor;
+@property (nonatomic, assign) BOOL readOnly;
+@property (nonatomic, assign) BOOL thumbnailViewEditingEnabled;
 
 - (void)initViewerSettings;
 - (void)applyViewerSettings;
 
+- (void)setAnnotationAuthor:(NSString *)annotationAuthor;
```

Comment:
Why only a setter method? Better to keep the property, and just use the methods as a "pass through" that references the source of true, the tool manager's property.

Reply:
We can discuss in meet or on xodo if you like.

Reply:
This is still true, this these should be properties, not directly declared as methods.